backend/app.ts

```diff
@@ -2,34 +2,55 @@ import * as dotenv from 'dotenv';
 import express from 'express';
 import cors from 'cors';
 import morgan from 'morgan';
-import session from 'express-session';
 import passport from 'passport';
 import googleOAuthRouter from './routes/googleOAuth';
 import testRouter from './routes/testRoute'
 import prisma from './utils/prismaClient';
-import loginAuth from './routes/loginAuth'
 import googleStrategy from 'passport-google-oauth20';
+import expressSession from 'express-session';
+import { PrismaSessionStore } from '@quixo3/prisma-session-store';
+import { PrismaClient } from '@prisma/client';
+import facebookStrategy from 'passport-facebook';
+import facebookOAuthRouter from './routes/facebookOAuth';
+import loginAuth from './routes/loginAuth'
 import { Strategy as LocalStrategy } from 'passport-local';
 import crypto from 'crypto';
+import session from 'express-session';
+import { secretcode, googleClientID, googleClientSecret, facebookAppSecret, facebookClientID } from './utils/config'
+import { request } from 'http';
+
 dotenv.config();
 const GoogleStrategy = googleStrategy.Strategy;
+const FacebookStrategy = facebookStrategy.Strategy;
 
-const db = prisma;
 const app = express();
+dotenv.config();
+const db = prisma;
 
 app.use(morgan("dev"));
 app.use(cors());
 app.use(express.json());
 
-const secretcode = process.env.SESSION_SECRET as string;
-const clientID = process.env.GOOGLE_CLIENT_ID as string;
-const clientSecret = process.env.GOOGLE_CLIENT_SECRET as string;
+app.use(
+    expressSession({
+        cookie: {
+            maxAge:  20 * 1000
```

Comment:
it's a good practice to do either of the following:

1) store a descriptive const variable with this number at the very top and use it below
e.g.
```
const HOURS_IN_MS = 60 * 60 * 1000
```

or

2) leave the duration in a comment
e.g.
```
maxAge: 20 * 1000 // 20 seconds
```